Modules/IO/GE/src/itkGE4ImageIO.cxx

```diff
@@ -191,7 +191,7 @@ GE4ImageIO::ReadHeader(const char * FileNameToRead)
     hdr->coordinateOrientation =
       itk::SpatialOrientationEnums::ValidCoordinateOrientations::ITK_COORDINATE_ORIENTATION_RSP;
   }
-  // RGEDEBUG(std::sprintf (debugbuf, "Plane = %d\n", hdr->imagePlane); cerr <<
```

Comment:
Thank you @seanm, but please consider simply removing those obsolete commented-out "RGEDEBUG" statements. I don't think they will ever be used again.

Reply:
Are you sure they are obsolete?  I don't use that class, and have no way to know if those debug statements are useful or not. The more conservative thing for me to do is as I've done.

Reply:
Thanks for your reply, @seanm This RGEDEBUG statement was commented out more than 20 years ago, with commit  cd3675f57ac94f85fba534e973f3190d87730696. So I guess it won't be "uncommented" back in the next 20 years to come.

The `RGEDEBUG` macro looks obsolete to me anyway, but that may be beyond the scope of your pull request. Your PR is specifically about commented out RGEDEBUG statements, if I understand correctly.

Reply:
>So I guess it won't be "uncommented" back in the next 20 years to come.

It might never be *committed* uncommented, but for all I know people use it every day *locally*.

>Your PR is specifically about...

I occasionally search my folder of third party code for "sprintf" in my mission to get rid of them all. I was tired of seeing this result in my searches. :)

Reply:
I second removing these statements.